card/mod/02_basic_types/spec/set/type/pointer_spec.rb

```diff
@@ -50,6 +50,93 @@
   end
 
 
+  let(:pointer) do
+    Card.create name: 'tp', type: 'pointer',
+                content: "[[item1]]\n[[item2]]"
+  end
+
+  describe '#added_item_names' do
+    it 'recognizes added items' do
+      Card::Auth.as_bot do
+        pointer
+        in_phase before: :approve,
+                 on: :save,
+                 trigger: ->{
+                   pointer.update_attributes!(
+                     content: "[[item1]]\n[[item2]]\n[[item3]]"
+                  )
+                } do
```

Comment:
`end` at 68, 16 is not aligned with `->{` at 64, 26 or `trigger: ->{` at 64, 17.